# Created at 2020-04-25T21:35:49.183
TestSet has not finished before stream error has appeared >> initializing exit by non-null configuration: DEFAULT
java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at org.apache.maven.surefire.booter.MasterProcessCommand.decode(MasterProcessCommand.java:115)
	at org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:391)
	at java.lang.Thread.run(Thread.java:745)


# Created at 2020-04-25T21:36:18.086
Unexpected IOException with stream: org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'RAHUL-PC', ip: '10.2.3.61', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_72'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output.
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:388:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'RAHUL-PC', ip: '10.2.3.61', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_72'
Driver info: driver.version: AndroidDriver

# Created at 2020-04-25T21:36:18.087
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.088
Unexpected IOException with stream: 	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)

# Created at 2020-04-25T21:36:18.089
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.089
Unexpected IOException with stream: 	at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)

# Created at 2020-04-25T21:36:18.090
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.090
Unexpected IOException with stream: 	at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)

# Created at 2020-04-25T21:36:18.091
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.091
Unexpected IOException with stream: 	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)

# Created at 2020-04-25T21:36:18.092
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.092
Unexpected IOException with stream: 	at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)

# Created at 2020-04-25T21:36:18.093
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.094
Unexpected IOException with stream: 	at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)

# Created at 2020-04-25T21:36:18.094
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.095
Unexpected IOException with stream: 	at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)

# Created at 2020-04-25T21:36:18.096
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.097
Unexpected IOException with stream: 	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)

# Created at 2020-04-25T21:36:18.098
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.098
Unexpected IOException with stream: 	at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)

# Created at 2020-04-25T21:36:18.099
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.099
Unexpected IOException with stream: 	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)

# Created at 2020-04-25T21:36:18.100
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.101
Unexpected IOException with stream: 	at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)

# Created at 2020-04-25T21:36:18.101
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.102
Unexpected IOException with stream: 	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:88)

# Created at 2020-04-25T21:36:18.102
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.103
Unexpected IOException with stream: 	at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)

# Created at 2020-04-25T21:36:18.104
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.105
Unexpected IOException with stream: 	at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)

# Created at 2020-04-25T21:36:18.106
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.107
Unexpected IOException with stream: 	at com.SIB.App.Base.Base.beforeTest(Base.java:69)

# Created at 2020-04-25T21:36:18.108
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.109
Unexpected IOException with stream: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

# Created at 2020-04-25T21:36:18.110
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.111
Unexpected IOException with stream: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

# Created at 2020-04-25T21:36:18.112
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.116
Unexpected IOException with stream: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

# Created at 2020-04-25T21:36:18.117
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.117
Unexpected IOException with stream: 	at java.lang.reflect.Method.invoke(Method.java:498)

# Created at 2020-04-25T21:36:18.118
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.119
Unexpected IOException with stream: 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)

# Created at 2020-04-25T21:36:18.119
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.120
Unexpected IOException with stream: 	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)

# Created at 2020-04-25T21:36:18.120
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.121
Unexpected IOException with stream: 	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)

# Created at 2020-04-25T21:36:18.121
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.122
Unexpected IOException with stream: 	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)

# Created at 2020-04-25T21:36:18.122
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.123
Unexpected IOException with stream: 	at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:619)

# Created at 2020-04-25T21:36:18.123
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.124
Unexpected IOException with stream: 	at org.testng.TestRunner.beforeRun(TestRunner.java:609)

# Created at 2020-04-25T21:36:18.125
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.125
Unexpected IOException with stream: 	at org.testng.TestRunner.run(TestRunner.java:580)

# Created at 2020-04-25T21:36:18.126
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.126
Unexpected IOException with stream: 	at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)

# Created at 2020-04-25T21:36:18.127
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.128
Unexpected IOException with stream: 	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)

# Created at 2020-04-25T21:36:18.128
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.129
Unexpected IOException with stream: 	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)

# Created at 2020-04-25T21:36:18.130
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.131
Unexpected IOException with stream: 	at org.testng.SuiteRunner.run(SuiteRunner.java:286)

# Created at 2020-04-25T21:36:18.132
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.133
Unexpected IOException with stream: 	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)

# Created at 2020-04-25T21:36:18.134
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.135
Unexpected IOException with stream: 	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)

# Created at 2020-04-25T21:36:18.135
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.136
Unexpected IOException with stream: 	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)

# Created at 2020-04-25T21:36:18.137
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.137
Unexpected IOException with stream: 	at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)

# Created at 2020-04-25T21:36:18.138
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.139
Unexpected IOException with stream: 	at org.testng.TestNG.runSuites(TestNG.java:1039)

# Created at 2020-04-25T21:36:18.140
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.140
Unexpected IOException with stream: 	at org.testng.TestNG.run(TestNG.java:1007)

# Created at 2020-04-25T21:36:18.141
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.142
Unexpected IOException with stream: 	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)

# Created at 2020-04-25T21:36:18.142
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.143
Unexpected IOException with stream: 	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)

# Created at 2020-04-25T21:36:18.143
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.144
Unexpected IOException with stream: 	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)

# Created at 2020-04-25T21:36:18.144
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.145
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)

# Created at 2020-04-25T21:36:18.145
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.146
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)

# Created at 2020-04-25T21:36:18.146
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.147
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)

# Created at 2020-04-25T21:36:18.147
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.148
Unexpected IOException with stream: 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

# Created at 2020-04-25T21:36:18.148
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.149
Unexpected IOException with stream: Caused by: java.lang.reflect.InvocationTargetException

# Created at 2020-04-25T21:36:18.149
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.150
Unexpected IOException with stream: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

# Created at 2020-04-25T21:36:18.150
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.151
Unexpected IOException with stream: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

# Created at 2020-04-25T21:36:18.153
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.154
Unexpected IOException with stream: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

# Created at 2020-04-25T21:36:18.155
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.155
Unexpected IOException with stream: 	at java.lang.reflect.Method.invoke(Method.java:498)

# Created at 2020-04-25T21:36:18.156
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.156
Unexpected IOException with stream: 	at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)

# Created at 2020-04-25T21:36:18.157
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.158
Unexpected IOException with stream: 	... 42 more

# Created at 2020-04-25T21:36:18.158
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.159
Unexpected IOException with stream: Caused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'RAHUL-PC', ip: '10.2.3.61', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_72'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output.
    at getResponseForW3CError (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Program Files\Appium\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:388:37)
    at process._tickCallback (internal/process/next_tick.js:68:7)

# Created at 2020-04-25T21:36:18.159
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.160
Unexpected IOException with stream: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)

# Created at 2020-04-25T21:36:18.161
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.161
Unexpected IOException with stream: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)

# Created at 2020-04-25T21:36:18.162
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.162
Unexpected IOException with stream: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)

# Created at 2020-04-25T21:36:18.163
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.163
Unexpected IOException with stream: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)

# Created at 2020-04-25T21:36:18.164
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.164
Unexpected IOException with stream: 	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)

# Created at 2020-04-25T21:36:18.165
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.165
Unexpected IOException with stream: 	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)

# Created at 2020-04-25T21:36:18.166
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.167
Unexpected IOException with stream: 	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)

# Created at 2020-04-25T21:36:18.168
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.168
Unexpected IOException with stream: 	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)

# Created at 2020-04-25T21:36:18.169
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.169
Unexpected IOException with stream: 	at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)

# Created at 2020-04-25T21:36:18.170
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.170
Unexpected IOException with stream: 	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)

# Created at 2020-04-25T21:36:18.171
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.172
Unexpected IOException with stream: 	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)

# Created at 2020-04-25T21:36:18.172
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.173
Unexpected IOException with stream: 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)

# Created at 2020-04-25T21:36:18.174
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.175
Unexpected IOException with stream: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)

# Created at 2020-04-25T21:36:18.175
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.176
Unexpected IOException with stream: 	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)

# Created at 2020-04-25T21:36:18.176
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.177
Unexpected IOException with stream: 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)

# Created at 2020-04-25T21:36:18.177
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.178
Unexpected IOException with stream: 	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)

# Created at 2020-04-25T21:36:18.178
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.179
Unexpected IOException with stream: 	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)

# Created at 2020-04-25T21:36:18.179
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:18.180
Unexpected IOException with stream: 	... 47 more

# Created at 2020-04-25T21:36:18.181
Unexpected IOException with stream: 


# Created at 2020-04-25T21:36:19.372
Unexpected IOException: 8,1,com.SIB.App.SanityTestCases.Declare_A_Disaster,initializePageObjects(com.SIB.App.SanityTestCases.Declare_A_Disaster),null,Can not set io.appium.java_client.MobileElement field com.SIB.App.Pages.LoginPage.accountName to org.openqa.selenium.remote.RemoteWebElement$$EnhancerByCGLIB$$d27c0df4,null,Can not set io.appium.java_client.MobileElement field com.SIB.App.Pages.LoginPage.accountName to org.openqa.selenium.remote.RemoteWebElement$$EnhancerByCGLIB$$d27c0df4,Declare_A_Disaster.initializePageObjects:45->Base.<init>:35 \00BB IllegalArgument ...,java.lang.IllegalArgumentException: Can not set io.appium.java_client.MobileElement field com.SIB.App.Pages.LoginPage.accountName to org.openqa.selenium.remote.RemoteWebElement$$EnhancerByCGLIB$$d27c0df4\000A\0009at com.SIB.App.SanityTestCases.Declare_A_Disaster.initializePageObjects(Declare_A_Disaster.java:45)\000A


# Created at 2020-04-25T21:36:19.375
Unexpected IOException: 5,1,com.SIB.App.SanityTestCases.Declare_A_Disaster,declareADisaster(com.SIB.App.SanityTestCases.Declare_A_Disaster),com.SIB.App.SanityTestCases.Declare_A_Disaster,null,null


# Created at 2020-04-25T21:36:19.376
Unexpected IOException: 9,1,com.SIB.App.SanityTestCases.Declare_A_Disaster,declareADisaster(com.SIB.App.SanityTestCases.Declare_A_Disaster),null,Can not set io.appium.java_client.MobileElement field com.SIB.App.Pages.LoginPage.accountName to org.openqa.selenium.remote.RemoteWebElement$$EnhancerByCGLIB$$d27c0df4,null


# Created at 2020-04-25T21:36:19.379
Unexpected IOException: 9,1,com.SIB.App.SanityTestCases.Declare_A_Disaster,afterTest(com.SIB.App.SanityTestCases.Declare_A_Disaster),null,null,null


# Created at 2020-04-25T21:36:20.338
Unexpected IOException: 2,1,org.apache.maven.surefire.testng.TestNGXmlTestSuite,TestSuite,null,null,null


